validators/bls.go

```diff
@@ -0,0 +1,227 @@
+package validators
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/fastrlp"
+)
+
+type BLSValidator struct {
+	Address      types.Address
+	BLSPublicKey []byte
+}
+
+func (v *BLSValidator) String() string {
+	return fmt.Sprintf(
+		"%s:%s",
```

Comment:
Where did you document this format with concatenation?

Reply:
Added a description above the method definition at 3128050498738a1c91d9f81e314574489904c841